docs/docs/configuration/semantic_search.md

```diff
@@ -0,0 +1,37 @@
+---
+id: semantic_search
+title: Using Semantic Search
+---
+
+Semantic search works by embedding images and/or text into a vector representation identified by numbers. Frigate has support for two such models which both run locally: [OpenAI CLIP](https://openai.com/research/clip) and [all-MiniLM-L6-v2](https://huggingface.co/sentence-transformers/all-MiniLM-L6-v2). Embeddings are then saved to a local instance of [ChromaDB](https://trychroma.com).
+
+## Configuration
+
+Semantic Search is a global configuration setting.
+
+```yaml
+semantic_search:
+  enabled: True
+```
+
+:::tip
+
+The embeddings database can be re-indexed from the existing detections in your database by adding an empty file named `.reindex` to the root of your configuration directory. Depending on the number of detections you have, it can take up to 30 minutes to complete and may max out your CPU while indexing.
```

Comment:
We do similar things with config values today. This might be better as a boolean config option for consistency.

Reply:
I can do that. Can you give me an example? My only worry is people forgetting to change it back to False or removing it. Can we automatically switch it back for them?

Reply:
Yea. I realized this after posting the comment that it would be important to not re-run. The parallel would be the `sync_recordings` setting under record. I am ok leaving it as is too.

Reply:
re-running is not really the end of the world, just takes extra time on large databases. I changed it locally, we can swap it back if we decide it's better, but adding a file to the config folder isn't the easiest thing on many installs.